source/index.markdown

```diff
@@ -27,8 +27,7 @@ These pages gives an overview of how to use Elastica. You can find the complete
 
 Compatibility
 --------------
-Elastica is tested with PHP 5.4 and later. Most functionality still works with PHP 5.3 but future support is not guaranteed. It Versions prior and equal to v0.19.8.0 are compatible with PHP 5.2
-
+Elastica is tested with PHP 7.0 and later. Most functionality still works with PHP 5.3 but future support is not guaranteed. It Versions prior and equal to v0.19.8.0 are compatible with PHP 5.2
```

Comment:
Sorry, misunderstand here. I would go with `Elastica 6.x is tested with PHP 7.0 and later.` and remove all the following content.

Reply:
Done :sweat_smile: